modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -3,16 +3,129 @@
+
+| `toc_depth`                         | `tableofcontents_depth`          | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+| `toc_header`                        | `tableofcontents_header`         | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+|===
+
+
+
 // tag::accompanying-premium-plugin-changes[]
 [[accompanying-premium-plugin-changes]]
 == Accompanying Premium Plugin changes
 
 The following premium plugin updates were released alongside {productname} 6.0.
 
+=== Enhanced Image Editing (was ImageTools)
+
+As noted in the xref:6.0-release-notes.adoc#removed-or-deprecated-plugins-imagetools[_{productname} 6.0 Release Notes_], the `imagetools` plugin is no longer included as part of the Free editor.
+
+_ImageTools_ is now _Enhanced Image Editing_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+NOTE: Enhanced Image Editing is the plugin’s name; `editimage` is the string used to reference theh plugin programmatically.
+
+Along with these changes, the `editimage_proxy` option was been renamed to `export_image_proxy_service_url`. The option was renamed to make it easier to configure when using https://tiny.cloud/docs/enterprise/server/[Tiny services].
+
+Finally, `editimage` also offers a new export plugin option: `export_image_proxy_service_url`.
+
+Users of the Free version of {productname} 5.x who rely on the, now deprecated and removed, `imagetools` plugin, should not upgrade to {productname} 6.0 without a https://tiny.cloud/pricing/[subscription plan] that provides access to Premium plugins such as https://tiny.cloud/tinymce/features/#productivity[_EditImage_].
+
+NOTE: `editimage_proxy` (the renamed equivalent to the previous `imagetools_proxy`) is still available as an alternative configuration option for setting up image proxy functionality without using Tiny services.
+
+
+=== SpellChecker Pro
+
+The free `spellchecker` plugin was removed as no longer supported:
+
+The https://tiny.cloud/tinymce/features/#productivity[Premium plugin], https://tiny.cloud/docs/plugins/premium/tinymcespellchecker/[Spellchecker Pro], which offers equivalent functionality and more, is available.
+
+
+=== PowerPaste
+
+As noted in the xref:6.0-release-notes.adoc#removed-or-deprecated-plugins-paste[_{productname} 6.0 Release Notes_], the `paste` plugin’s functionality is no longer provided as a plugin. It is now a core part of {productname} 6.0.
+
+Because the functionality was unmaintainable, however, `paste` no longer supports input from Microsoft Word.
+
+The Premium plugin, https://tiny.cloud/docs/plugins/premium/powerpaste/[`PowerPaste`], is now available. It provides the capability to accept data from Microsoft Word and Microsoft Excel, and clean-up the received data before pasting it into place.
```

Comment:
I do kind of like having this here, since it's pointing people to PowerPaste. I think it either needs a subheading to explain that it's about core paste though, or remove it (and maybe we can figure it out later)?

Reply:
I'd like to reword it a bit - that makes it sound like powerpaste is new. It's not, that is just the recommendation to use now that `paste` deleted word support.